pyopencl/cl/pyopencl-complex.h

```diff
@@ -274,11 +274,20 @@
     } \
   } \
 
+// This is undocumented and may disappear at any time
+#if PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT
```

Comment:
```suggestion
#ifdef PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT
```
?

Reply:
I wanted to do
```
#if defined(PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT) and PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT
```
to distinguish `#define PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT 0` and `#define PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT 1`,
but the above expression is equal to `#if PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT`.

Reply:
OK, fair.